src/main/perl/Path.pm

```diff
@@ -544,6 +566,180 @@ sub directory
 }
 
 
+=item _make_link
+
+This method is mainly a wrapper over C<LC::Check::link>
+returning the standard C<CAF::Path> return values. Every option
+supported by C<LC::Check::link> is supported. C<NoAction>
+flag is handled by C<LC::Check::link> and C<keeps_state> option
+is honored (overrides C<NoAction> if true). One important
+difference is the order of the arguments: C<CAF::Path:_make_link>
+and the methods based on it are following the Perl C<symlink>
+(and Shell C<ln>) argument order.
```

Comment:
`ln` has nothing to do with `Shell`, it's part of base utils on a lot of Unix systems.